src/main/resources/application.yaml

```
@@ -0,0 +1,63 @@
+    access:
+      prefix: Authorization
+      secret: ${JWT_ACCESS_SECRET:${JWT_ACCESS_SECRET_DEFAULT}}
+      expiration-ms: 900000      # 15?
+    refresh:
+      prefix: refreshToken
+      secret: ${JWT_REFRESH_SECRET:${JWT_REFRESH_SECRET_DEFAULT}}
+      expiration-ms: 604800000   # 7?
+
+cors:
+  allowed-origins: >
+    {FE_URL}, {BE_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)